src/App/Fossa/Main.hs

```diff
@@ -319,8 +319,9 @@ testOpts =
     <*> baseDirArg
 
 vpsOpts :: Parser VPSOptions
-vpsOpts = VPSOptions <$> skipIprScanOpt <*> licenseOnlyScanOpt <*> fileFilterOpt <*> vpsCommands
+vpsOpts = VPSOptions <$> followSymlinksOpt <*> skipIprScanOpt <*> licenseOnlyScanOpt <*> fileFilterOpt <*> vpsCommands
   where
+    followSymlinksOpt = flagOpt FollowSymlinks (long "follow" <> help "If specified, follows symbolic links but has no protection against loops")
```

Comment:
[suggestion]: I think a slight rewording might read better, but it's up to you.
```suggestion
    followSymlinksOpt = flagOpt FollowSymlinks (long "follow" <> help "If specified, follows symbolic links (does not protect against cyclic links)")
```

Also, I think that we may want to re-word all of these (in a future PR) to not include the phrase "If specified".  I prefer reading these options in the imperative tone, rather than declarative.  e.g. "Follow symbolic links (does not protect against cyclic links)"

Reply:
Agreed about the imperative tone. I'll reword.